modules/gapi/include/opencv2/gapi/gcomputation.hpp

```diff
@@ -286,7 +289,7 @@ class GAPI_EXPORTS_W GComputation
      * @param args compilation arguments for underlying compilation
      * process.
      */
-    GAPI_WRAP void apply(cv::Mat in, CV_OUT cv::Scalar &out, GCompileArgs &&args = {}); // Unary overload (scalar)
```

Comment:
This is no longer necessary, because only overload with `GRunArgs` should be used from python

But `comp.apply(cv.gin(in1, in2, in3), args=cv.compile_args(pkg))` looks ugly. 
I believe, it will be `comp.apply(in1,in2,in3, args=cv.compile_args(pkg))` in next PR's